src/index.ts

```diff
@@ -1,12 +1,14 @@
 // Copyright IBM Corp. 2017. All Rights Reserved.
-// Node module: @loopback/loopback-next-hello-world
+// Node module: loopback-next-hello-world
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 'use strict';
 
 import {CoreBindings} from '@loopback/core';
 import {HelloWorldApp} from './application';
 
+export {HelloWorldApp} from './application';
```

Comment:
Since we have already imported `HelloWorldApp`, I think it's enough to just export it?

```ts
export {HelloWorldApp};
// or maybe
export HelloWorldApp;
```